app/main/views/g_cloud.py

```diff
@@ -66,9 +67,14 @@ def get_service_by_id(service_id):
             abort(404, "Service ID '{}' can not be found".format(service_id))
 
         service_data = service['services']
+        framework_slug = service_data['frameworkSlug']
+        override_framework_slug = current_app.config.get('DM_FRAMEWORK_CONTENT_MAP').get(framework_slug)
```

Comment:
Again, I think a short comment here to explain why it's overriding anything wouldn't hurt.